CWAs/CWA-2024-009.md

```diff
@@ -6,11 +6,13 @@ Low (Marginal + Likely)[^1]
 
 **Affected versions:**
 
-- wasmd < 0.53.1
+- wasmd > 0.34.1, < 0.53.1
```

Comment:
This range marks future releases such as 0.34.2 as affected. I think this is what we want:

```suggestion
- wasmd > 0.40.0, < 0.53.1
```

Reply:
Ah, nice catch.